search/settings.py

```diff
@@ -0,0 +1,124 @@
+"""
+Django settings for search project.
+
+Generated by 'django-admin startproject' using Django 3.0.4.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/3.0/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/3.0/ref/settings/
+"""
+
+import os
+from dotenv import load_dotenv
+
+# reading .env file
+load_dotenv()
+
+# Build paths inside the project like this: os.path.join(BASE_DIR, ...)
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/3.0/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = os.getenv('SECRET_KEY')
+
+# SECURITY WARNING: don't run with debug turned on in production!
+DEBUG = os.getenv('DJANGO_DEBUG')
+
+ALLOWED_HOSTS = ['localhost', '127.0.0.1', 'place-search.herokuapp.com']
```

Comment:
This should be among your env variables so that it doesn't get exposed

Reply:
But the deployed app is not private na!
The url is on my repo description already